src/Utils.php

```diff
@@ -231,7 +231,7 @@ public static function encodeURL($url) {
 		// $url = str_replace(")", "%29", $url);
 		// In fact, you even have to *undo* some encodings from MW's Title::getFullURL()
 		$url = preg_replace_callback(
-			"/(?<=[^%])%(?:21|25|27|28|29|2a)/i",
+			"/(?<=[^%])%(?:21|27|28|29|2a)/i",
```

Comment:
Unfortunately, there doesn't seem to be a way to support links with literal percents in them.

* `https://runescape.wiki/w/50%25_Luke` results in a non-hyperlink.
* `https://runescape.wiki/w/50%_Luke` results in a hyperlink, but clicking it opens a modal "<...> is malformed and potentially dangerous." with no way to proceed.

The first seems the better of the two options, hence the removal of `25` from the decoding here.